src/core/lib/channel/channelz.h

```diff
@@ -262,7 +262,8 @@ class SocketNode : public BaseNode {
   gpr_atm last_remote_stream_created_millis_ = 0;
   gpr_atm last_message_sent_millis_ = 0;
   gpr_atm last_message_received_millis_ = 0;
-  UniquePtr<char> peer_string_;
+  UniquePtr<char> remote_host_;
```

Comment:
Shouldn't this also track the local address?

Reply:
It should, I haven't gotten to that yet. Still looking for the best place to pick that up from the code